PA-13458 Add Docs for different ways of scanning with csa (#22)
soos-io/kb-docs: CSA/Script.md

Note: The path c:/results is specific to Windows. If you're using Linux or macOS, adjust the path format accordingly.

---

## Scanning a Locally Built Docker Image

To scan a Docker image that has been built locally and is not hosted on any registry, follow these steps:

> 1. Give access to Docker Host Context to the `soosio/sca` image. This is done by mounting the Docker socket from your host machine into the container.

Reviewer: ```suggestion
1. Give access to Docker Host Context to the `soosio/sca` image. This is done by mounting the Docker socket that the host listens on from the host machine to the container.
```

   --projectName="<YOUR_PROJECT_NAME>" \
   <PREVIOUSLY_BUILT_IMAGE>
   ```

Replace `<PREVIOUSLY_BUILT_IMAGE>` with the name of your locally built Docker image. This command enables the scanning tool inside the container to access and scan the specified Docker image stored on your host machine.

---

## Scanning a Docker Image Saved as a .tar File

To scan a Docker image that has been saved as a `.tar` file, follow these steps:

1. Save the Docker Image: Use the `docker save` command to save your image as a `.tar` file. For example, to save the `soosio/csa` image, use:
   ```
   docker save -o soosio_csa.tar soosio/csa
   ```

2. Run the Scan: Execute the following command, ensuring that the volume is mounted to the path where the `.tar` file is located:
   ```
   docker run -it \
   -v <YOUR_LOCAL_DIRECTORY_PATH>:/usr/src/app/results:rw \
   soosio/csa \
   --clientId=<YOUR_CLIENT_ID> \
   --apiKey=<YOUR_API_KEY> \
   --projectName="<YOUR_PROJECT_NAME>" \
   "docker-archive:results/your_image.tar"
   ```

Replace `<YOUR_LOCAL_DIRECTORY_PATH>` with the directory containing your `.tar` file. This command mounts the `.tar` file inside the container and initiates the scan.

> ## Scanning a Directory

Reviewer: ```suggestion
## Scanning a Docker Image Definition Directory
```

> To scan a directory containing Docker images or other relevant files, use the following method:

Reviewer: ```suggestion
To scan a directory containing Docker image definitions with its other relevant files, use the following method:
```
